cypress/e2e/immutableDatabaseTests/workflowDetails.ts

```diff
@@ -141,3 +141,23 @@ describe('Test primary descriptor bubble', () => {
     cy.get('[data-cy=primary-descriptor-bubble]').should('be.visible');
   });
 });
+
+describe.only('Test engine versions', () => {
```

Comment:
Should remove `.only`

Reply:
🤦 Thanks for the catch! I keep making that mistake.